guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -17,6 +16,31 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## v2002.0.22
+
+-  {:.new}**Changes to the ece-tools composer package**
```

Comment:
Can we please highlight this somehow, so it's better visible VS a regular release line? Something like info block would work.